yojimbo.cpp

```diff
@@ -669,7 +669,7 @@ namespace yojimbo
 
 static void default_assert_handler( const char * condition, const char * function, const char * file, int line )
 {
-    printf( "assert failed: ( %s ), function %s, file %s, line %d\n", condition, function, file, line );
+    yojimbo_printf( YOJIMBO_LOG_LEVEL_NONE, "assert failed: ( %s ), function %s, file %s, line %d\n", condition, function, file, line );
```

Comment:
Why not LOG_LEVEL_ERROR for an assertion failure?

Reply:
Because `YOJIMBO_LOG_LEVEL_NONE` is less than `YOJIMBO_LOG_LEVEL_ERROR`, so even if you suppress errors (by setting `yojimbo_log_level(YOJIMBO_LOG_LEVEL_NONE)`, the messages will log. The other alternative would be to call `printf_function` directly, rather than going through the `yojimbo_printf` macro.

Reply:
Good call.